WireUI/Package.swift

```diff
@@ -0,0 +1,58 @@
+// swift-tools-version: 5.10
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "WireUI",
+    defaultLocalization: "en",
+    platforms: [
+        .iOS(.v15),
+        .macOS(.v12)
+    ],
+    products: [
+        .library(
+            name: "WireDesign",
+            targets: ["WireDesign"]),
+        .library(
+            name: "WireReusableUIComponents",
+            targets: ["WireReusableUIComponents"])
+    ],
+    dependencies: [
+        .package(
+            url: "https://github.com/apple/swift-docc-plugin",
+            from: "1.1.0"
+        ),
+        .package(
+            url: "https://github.com/pointfreeco/swift-snapshot-testing",
+            from: "1.16.0"
+        )
+    ],
+    targets: [
+
+        .target(
+            name: "WireDesign"),
+        .testTarget(
+            name: "WireDesignTests",
+            dependencies: [
+                "WireDesign",
+                .product(
+                    name: "SnapshotTesting",
+                    package: "swift-snapshot-testing"
+                )
+            ]),
+
+        .target(
+            name: "WireReusableUIComponents",
+            dependencies: ["WireDesign"]),
+        .testTarget(
+            name: "WireReusableUIComponentsTests",
```

Comment:
what's the difference between WireDesign and WireReusableUIComponents ?

Reply:
what about adding docC to present the package at the same time?

Reply:
> what's the difference between WireDesign and WireReusableUIComponents ?

`WireDesign` will have everything around colors, fonts, styles.
We discussed to have targets like `ConversationList`, `UserProfile` etc.

I created a component which displays the [user account image and the availability status](https://github.com/wireapp/wire-ios/pull/1583) and since that component does not only belong into `ConversationList`, I created another target named `WireReusableUIComponents`, for shared UI components.

Reply:
> what about adding docC to present the package at the same time?

sure, will add it